src/qml/controls/ExternalLink.qml

```diff
@@ -46,4 +46,17 @@ Control {
             }
         }
     }
+
+    background: Rectangle {
+        visible: root.visualFocus
+        anchors.fill: parent
+        anchors.leftMargin: -15
+        anchors.rightMargin: -4
+        anchors.topMargin: -4
+        anchors.bottomMargin: -5
```

Comment:
Why have different values like here? The border should be nicely defined and uniform, it is the elements themselves that differ in how much space they take up, and we need to address those issues in follow-ups. 

I would keep the definitions in all of these to just specify `anchors.margins: -4` and address the issues in the elements themselves, instead of changing how the border works to accommodate the size issues.

It might be possible to extrapolate this out into our theme file, but we can leave that for much later.

Reply:
Ok, I like that approach. I'll try it out and see if it can be adjusted properly with margins/padding values on the elements.

Reply:
I'm going to ACK this

this pr doesn't have to address the issues with the elements, only to keep the declaration of the margins of the border to only `anchors.margins: -4` for now, alternatively we can merge like this and address the issues in a follow-up and clean up the definition of the border.